sfge/src/test/java/com/salesforce/graph/ops/MethodUtilTest.java

```diff
@@ -312,4 +317,59 @@ public void getTargetMethods_targetMethodInInnerAndOuterClass() {
                 messages,
                 containsString(EventKey.WARNING_MULTIPLE_METHOD_TARGET_MATCHES.getMessageKey()));
     }
+
+    @ValueSource(strings = {Schema.AURA_ENABLED, Schema.NAMESPACE_ACCESSIBLE})
```

Comment:
This is a parameterized refactor of the aura-enabled test in the other test file, also covering NamespaceAccessible and any other annotations we add support for.

Reply:
Can we also have a test at a higher level to see if NameAccessible annotations are recognized as entry points?

Reply:
Probably. I'll add that first thing tomorrow.